pkg/api/v1/router_bom_test.go

```diff
@@ -0,0 +1,304 @@
+package serverservice_test
+
+import (
+	"context"
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+
+	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+)
+
+func cleanupDB() error {
```

Comment:
the mock server initialises the [test database](https://github.com/metal-toolbox/hollow-serverservice/blob/a4fb59f85b508e3eb960f540a6d158705bceeb93/pkg/api/v1/router_int_test.go#L29)  and register a testing.T [cleanup handler](https://github.com/metal-toolbox/hollow-serverservice/blob/a4fb59f85b508e3eb960f540a6d158705bceeb93/internal/dbtools/testtools.go#L97) can you add the Delete into that cleanup handler instead?

Reply:
Thanks so much for the link and info! That's really helpful and optimized the tests code there.